b2sdk/raw_simulator.py

```diff
@@ -591,6 +592,7 @@ def upload_file(
         self.file_name_and_id_to_file[file_sim.sort_key()] = file_sim
         return file_sim.as_upload_result()
 
+    @set_token_type(TokenType.UPLOAD_PART)
```

Comment:
this is for a signature of a method of `BucketSimulator` class, which is not a part of `RawApi` family, but is a helper for holding data for `RawSimulator`, which is a part of `RawApi` family. Only the methods of `RawApi` family need this decorator, therefore please remove the decorators from here and one place `BucketSimulator.upload_file()` above